src/ngraph/frontend/onnx_import/op/dequantize_linear.cpp

```diff
@@ -81,6 +78,19 @@ namespace ngraph
                                                                            x->get_element_type());
                     }
 
+                    Shape zero_point_shape = zero_point->get_shape();
+                    if (zero_point_shape.size() == 1 && zero_point_shape[0] == 1)
```

Comment:
We are repeating this block of code many times. Let's make a small helper function called `reshape_to_scalar_if_possible` and just call that many times.

Reply:
You could even try to pass it a `NodeVector` to reduce the number of calls.